.github/workflows/publish-conductor.yml

```diff
@@ -5,13 +5,14 @@ on:
       - main
   release:
     types: [published]
+  workflow_dispatch:
 
 # Defines two custom environment variables for the workflow. These are
 # used for the Container registry domain, and a name for the Docker
 # image that this workflow builds.
 env:
   REGISTRY: ghcr.io
-  IMAGE_NAME: FAIMS/faims3-api
+  IMAGE_NAME: nbic-bss/faims3-api
```

Comment:
this also shouldn't be here

Reply:
We should make this customisable via variables.

Reply:
Customised via ${{github.repository_owner}}.